src/Host/Protocol/Impl/CustomHttpError.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.R.Host.Protocol {
+    public enum CustomHttpError {
```

Comment:
BTW nothing prevents us from inventing our own range, this is a not a public server anyway...